index.js

```diff
@@ -375,6 +375,144 @@ BabelClient.prototype.createAnnotation = function(token, data, options, callback
 };
 
 /**
+ * Update an annotation
+ *
+ * @param {string} token Persona token
+ * @param {object} data Data that can be passed into an annotation
+ * @param {object} data._id
+ * @param {object} data.hasbody
+ * @param {string} data.hasBody.format
+ * @param {string} data.hasBody.type
+ * @param {string} data.hasBody.chars
+ * @param {object} data.hasBody.details
+ * @param {string} data.hasBody.uri
+ * @param {string} data.hasBody.asReferencedBy
+ * @param {object} data.hasTarget
+ * @param {string} data.hasTarget.uri
+ * @param {string} data.hasTarget.fragment
+ * @param {string} data.hasTarget.asReferencedBy
+ * @param {string} data.annotatedBy
+ * @param {string} data.motiviatedBy
+ * @param {string} data.annotatedAt
+ * @param callback
+ */
+BabelClient.prototype.updateAnnotation = function(token, data, callback){
+    if (!token) {
+        throw new Error('Missing Persona token');
+    }
+    if (!data._id) {
+        throw new Error('Missing data: _id');
+    }
+    if (!data.hasBody) {
+        throw new Error('Missing data: hasBody');
+    }
+    if (!data.hasBody.format) {
+        throw new Error('Missing data: hasBody.format');
+    }
+    if (!data.hasBody.type) {
+        throw new Error('Missing data: hasBody.type');
+    }
+    if (!data.annotatedBy) {
+        throw new Error('Missing data: annotatedBy');
+    }
+    if (!data.hasTarget) {
+        throw new Error('Missing data: hasTarget');
+    }
+
+    // validate the hasTarget property
+    var targets = [];
+    if (_.isArray(data.hasTarget)) {
+        targets = data.hasTarget;
+        if (targets.length === 0) {
+            throw new Error('Missing data: hasTarget cannot be empty array');
+        }
+    } else {
+        targets.push(data.hasTarget);
+    }
+    _.map(targets,function(target) {
+        if (!_.has(target, 'uri')) {
+            throw new Error('Missing data: hasTarget.uri is required');
+        }
+        for (var prop in target) {
+            if (!(prop === 'uri' || prop === 'fragment' || prop === 'asReferencedBy' )) {
+                throw new Error('Invalid data: hasTarget has unrecognised property \'' + prop + '\'');
+            }
+        }
+    });
+
+    var requestOptions = {
+        method: 'PUT',
+        body: data,
+        json: true,
+        url: this.config.babel_host + ':' + this.config.babel_port + '/annotations/' + data._id,
+        headers: {
+            'Accept': 'application/json',
+            'Authorization': 'Bearer ' + token,
+            'Host': this.config.babel_hostname
+        }
+    };
+
+    this.debug(JSON.stringify(requestOptions));
```

Comment:
can you give a example of the issue? Persona (when the branch is merged) will hold a audit of all tokens created within the last 15 days. We could output the JWT ID instead of the token. Even better would be to output the body of the JWT token without the signature.